src/Gaufrette/Adapter/InMemory.php

```diff
@@ -53,7 +53,7 @@ public function setFiles(array $files)
      * @param string $content The content
      * @param int    $mtime   The last modified time (automatically set to now if NULL)
      */
-    public function setFile($key, $content = null, $mtime = null)
+    public function setFile(string $key, string $content = null, int $mtime = null): void
```

Comment:
```suggestion
    public function setFile(string $key, ?string $content = null, ?int $mtime = null): void
```

Reply:
we will used a tool to add this, once again the work is only focus on typehint :slightly_smiling_face:

Reply:
I don't agree with @Zusoy, why allowing null type for the `$mtime` argument ?